onecodex/viz/_distance.py

```diff
@@ -185,7 +198,21 @@ def plot_distance(
 
             formatted_fields.append(field_group)
 
-        clust = self._cluster_by_sample(rank=rank, metric=metric, linkage=linkage)
+        clust = self._cluster_by_sample(
+            rank=rank,
+            metric=metric,
+            linkage=linkage,
+            all_nan_classification_ids=self._all_nan_classification_ids,
+            exclude_all_nan=True,
+        )
+
+        if self._all_nan_classification_ids:
+            warnings.warn(
```

Comment:
This doesn't raise a warning for me. The first arg to `warnings.warn` should be the message and the second arg is the warning type:

```python
warnings.warn(
    f"{len(self._all_nan_classification_ids)} sample(s) have no abundances calculated and have been omitted from the distance heatmap.",
    PlottingWarning,
)
```